features/FEATURE_PSA/supporting_psa_in_mbed-os.md

```diff
@@ -120,8 +183,36 @@ example mbed_app.json:
 }
 ```
 
+# Building TF-M
+On dual v7-M and v8-M targets, TF-M will be running on SPE providing PSA services. A python script `tools/psa/build_tfm.py` automates building TF-M and copying the binary to predefined location defined by attribute `delivery_dir`.
 
-## New directory structure
+Usage:
+```console
+python3 tools/psa/build_tfm.py
+```
+Supported options:
+```console
+usage: build_tfm.py [-h] [-m {CY8CPROTO_064_SB_S}] [-t {ARMCLANG,GNUARM}] [-d]
+                    [-l] [--clone] [--commit] [--clean-build] [-v]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -m {CY8CPROTO_064_SB_S}, --mcu {CY8CPROTO_064_SB_S}
+                        Build for the given MCU
+  -t {ARMCLANG,GNUARM}, --toolchain {ARMCLANG,GNUARM}
+                        Build for the given tool chain (default is
```

Comment:
No space in "tool chain" please, here and in the Python script.

Reply:
Fixed